crates/store/re_types/definitions/rerun/blueprint/views/dataframe.fbs

```diff
@@ -0,0 +1,13 @@
+namespace rerun.blueprint.views;
+
+/// A view to display any data in a tabular form.
+///
```

Comment:
```suggestion
/// A view to display any data in a tabular form.
///
/// Data can be shown with arbitrary queries, both at single timepoints or over ranges each of which
/// may or may not be dependent on the current time selection.
```

.. something like that, to expand on it a bit more? :)

Reply:
Totally. I'll made a note to fix that in part 1